build/tools/Automation/Constants.cs

```diff
@@ -1,25 +1,14 @@
-// --------------------------------------------------------------------------------------------
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT license.
-// --------------------------------------------------------------------------------------------
```

Comment:
Could you please clarify if there are any reasons to remove the MIT license header here? Looks like it should still be valid in that case.

Reply:
Sorry, I accidentally deleted. But have added back